xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-api/src/test/java/org/xwiki/livedata/internal/LiveDataRendererConfigurationTest.java

```diff
@@ -19,6 +19,8 @@
  */
 package org.xwiki.livedata.internal;
 
+import java.util.List;
```

Comment:
Surprising to see this import change alone

Reply:
Probably a mistake, checking to remove it.